packages/paste-core/components/form-pill-group/src/PillCloseIcon.tsx

```diff
@@ -49,8 +52,8 @@ export const PillCloseIcon: React.FC<React.PropsWithChildren<PillCloseIconProps>
       display="flex"
       alignItems="center"
       justifyContent="center"
-      height="sizeIcon40"
-      width="sizeIcon40"
+      height={size === "large" ? "sizeIcon50" : "sizeIcon40"}
+      width={size === "large" ? "sizeIcon50" : "sizeIcon40"}
```

Comment:
**issue:** PillCloseIcon for the large variant should be slightly bigger. Currently, both variants seem to be the same size.
![Screenshot 2024-07-26 at 2 29 51 PM](https://github.com/user-attachments/assets/c06269ad-789b-4036-a277-2fe4af234abf)

Design sizes:

Default (PillCloseIcon): size-icon-10
Large (PillCloseIcon): size-icon-20

Reply:
Note: Not sure why the sizes are different in the code :'( is that expected? or am I checking in the wrong place?

Reply:
It's on line 63 of this file 🤓 I'll bump it up to sizeIcon20